### DISCUSSION TITLE
samadrit das <>Resume Review

### DISCUSSION THREAD

**Discussion by THUNDERSAMA** [recodehive/resume-pitch#3] [Q&A] [answered]
samadrit das

https://www.linkedin.com/in/samadrit-das/

[samadrit RESUME.pdf](https://github.com/user-attachments/files/16400693/samadrit.RESUME.pdf)

https://github.com/THUNDERSAMA

I am now a 4th-year student studying in a tier 3 college, how can I make my resume good so that I can apply to a company off-campus, looking for a full stack developer position, and how to improve my ATS score also I have applied to many company outsides for an intern position but never got a reply, could you help me with that

https://samadrit-das.netlify.app/

**Comment by sanjay-kv** [comment, marked as answer]
@THUNDERSAMA Sorry, i was packed with the gssoc program. i will review by EOd

**Reply by sanjay-kv** [reply]
[sam.pdf](https://github.com/user-attachments/files/16580063/sam.pdf)
[ATS Version of CV (10).docx](https://github.com/user-attachments/files/16580065/ATS.Version.of.CV.10.docx)

**Reply by sanjay-kv** [reply]
https://www.loom.com/share/18479c26aea849c7a281e5ce76c36661?sid=496a7f17-79a2-430c-b326-1e90f92f1634
